server/data/formClient.js

```diff
@@ -20,15 +20,16 @@ const selectClause = `select id,
                     review_reason          as "reviewReason",
                     nomis_sequence_no      as "nomisSeq"`
 
-const sequenceClause =
-  'and f.sequence_no = (select max(f2.sequence_no) from form f2 where f2.booking_id = f.booking_id)'
-
 const ignoreCancelledClause = `and f.status <> 'CANCELLED'`
 
+const sequenceClause = `and f.sequence_no = (select max(f2.sequence_no) from form f2 where f2.booking_id = f.booking_id and f2.status <> 'CANCELLED')`
+
+const sequenceClauseIncludeCancelled = `and f.sequence_no = (select max(f2.sequence_no) from form f2 where f2.booking_id = f.booking_id)`
```

Comment:
left this in because some queries are by status and it didn't make sense to also restrict by status